app/components/render-node.tsx

```
@@ -0,0 +1,203 @@
+  root = false,
+}: {
+  node: WikiNode;
+  root?: boolean;
+}) {
+  switch (node.type) {
+    case NodeType.Section:
+      return (
+        <Fragment>
+          {!root && node.properties.title && (
+            <Markdown
+              children={
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)